tests/client-sdk/inference/test_inference.py

```diff
@@ -69,6 +72,16 @@ def get_weather_tool_definition():
     }
 
 
+@pytest.fixture
+def base64_image_url():
+    downloadable_url = "https://www.healthypawspetinsurance.com/Images/V3/DogAndPuppyInsurance/Dog_CTA_Desktop_HeroImage.jpg"
```

Comment:
can you embed this (or some other) jpg into the repository and use its content as the base64? that way the test does not need to download the URL.

Reply:
done